fawltydeps/settings.py

```diff
@@ -98,6 +98,47 @@ def parse_path_or_stdin(arg: str) -> PathOrSpecial:
     return Path(arg)
 
 
+DEFAULT_IGNORE_UNUSED = {
+    # Development tools not meant to be imported
+    # Formatting Tools
+    "autopep8",
+    "black",
+    "isort",
+    "pyformat",
+    "yapf",
+    # Linting Tools
+    "bandit",
+    "flake8",
+    "mccabe",
+    "pyflakes",
+    "pylint",
+    # Security Tools
+    "codespell",
+    # Documentation Tools
+    "sphinx",
+    "sphinx-rtd-theme",
```

Comment:
Not expected to be imported, but also not a "tool" in its own right (i.e. it does not include a command-line interface). Maybe more of a "tool plugin" in the same category as `pytest-*` packages, I guess?

I'm also guessing there are more sphinx themes than just this one that are in common use?

In general, do we want to consider a more general solution to these categories of plugin packages? Maybe allowing simple globs like `pytest-*` and `sphinx-*` in this set? (although this will add complexity to our matching algorithm in `fawltydeps/check.py`).

But I'm also very happy to leave this as-is for now. We can always reconsider later if/when we see that more of these packages need to be added.

Reply:
Agreed. 
I've excluded `sphinx-*` in https://github.com/tweag/FawltyDeps/pull/365/commits/f3408b80582e70cb7ae66e589b9774b12ce62c4b. We may go back and allow `pytest-*` and `sphinx-*` in this set later on.